troubleshoot/index.md

```diff
@@ -57,14 +57,48 @@ If you have an [Elastic subscription](https://www.elastic.co/pricing), you can c
   If you contact us by email, use the email address you registered with so we can help you more quickly. If your registered email is a distribution list, you can register a second email address with us. Just open a case to let us know the name and email address you want to add.
   :::
 
+  :::{warning}
+  All cases opened by email default to normal severity. For incidents, open a case through the [Elastic Support Portal](https://support.elastic.co/) and select the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+  :::
+
 ## Working with support [troubleshoot-work-with-support]
 
-Try these tips when opening a support case:
+When you open a support case:
+
+* Describe the problem and its [contextual severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+
+  * If an error message was encountered, include full error message and timezone-designated dates and times of when the problem occurred.
+
+  * If the problem is UI-related and does not replicate via API, pull a [browser network log](https://www.elastic.co/blog/generating-browser-har-file-kibana-troubleshooting). 
+
+* Upload the related product's diagnostics and debug logs
+
+  * If hosting on {{serverless-full}} or {{ech}}, Support can pull diagnostics on your behalf as long as you include resource IDs:
+    
+    * {{serverless-full}} Project ID or {{kib}} URL
+    * {{ech}} Deployment ID or {{kib}} URL
 
-* Include the deployment ID that you want help with, especially if you have several deployments.
+  * If hosting on {{ece}} or {{eck}}:
 
-  You can find the deployment ID on the overview page for your cluster in the {{ecloud}} Console.
+    * {{ece}} [diagnostic](/troubleshoot/deployments/cloud-enterprise/run-ece-diagnostics-tool.md) flagging `--deployments` as applicable
+    * {{eck}} [diagnostic](/troubleshoot/deployments/cloud-on-k8s/run-eck-diagnostics.md)
 
-* Describe the problem. Include any relevant details, including error messages you encountered, dates and times when the problem occurred, or anything else you think might be helpful.
+  * If self-hosting:
+
+    * {{es}} [diagnostic](troubleshoot/elasticsearch/diagnostic.md) and [debug logs](deploy-manage/monitor/logging-configuration/update-elasticsearch-logging-levels.md)
+    * {{kib}} and {{fleet}} [diagnostic](troubleshoot/kibana/capturing-diagnostics.md) and [debug logs](/docs/deploy-manage/monitor/logging-configuration/kibana-log-levels.md)
+    
+    * Ingest tools
+
+      * {{ls}} [diagnostic](troubleshoot/ingest/logstash/diagnostic.md) and [debug logs](reference/logstash/logging.md)
```

Comment:
```suggestion
      * {{ls}} [diagnostic](/troubleshoot/ingest/logstash/diagnostic.md) and [debug logs](docs-content://logging.md)
```
This should fix the first one. I'm not sure about the logstash one but let's see if this works. :-)